hadoop-project/pom.xml

```diff
@@ -92,7 +92,7 @@
     <spotbugs.version>3.1.0-RC1</spotbugs.version>
     <dnsjava.version>2.1.7</dnsjava.version>
 
-    <guava.version>11.0.2</guava.version>
+    <guava.version>27.0-jre</guava.version>
```

Comment:
The latest available Guava version is `27.1-jre`. Is there a reason why `27.0-jre` was chosen instead?

Reply:
we decided to update a while ago with other components to 27.0-jre
it's easier to do the upgrade, 27.1-jre includes many other deprecations which would be way harder to do from 11.0.2 - so this is easier, precommit jenkins won't time out. If we really need to update 27.1-jre we can do that after fixing the new deprecations: https://issues.apache.org/jira/browse/HADOOP-16222